docs/developer_guide.md

```diff
@@ -104,6 +104,36 @@ opensearch-data-prepper:1.2.0-SNAPSHOT
 ```
 
 
+## Contributing your Code to Data Prepper
+
+When you wish to contribute a change to Data Prepper, please create a GitHub Pull Request (PR). You should create your PR to target `main`.
+
+The Data Prepper maintainers will review your PR and merge it once it is approved.
+
+Some changes contain bug fixes or security fixes. If you believe your
```

Comment:
take it or leave it: I would recommend the call to action to be definite. "read on" is open ended.
```
Some changes containing bug fixes or security fixes may be eligible for a patch release. If you believe your change should be a patch release, please see [Backporting](#Backporting)
```